keras_cv/utils/fill_utils.py

```diff
@@ -33,24 +33,28 @@ def rectangle_masks(corners, mask_shape):
     corners = corners[..., tf.newaxis, tf.newaxis]
 
     # split coordinates
-    x0 = corners[:, 0]
-    y0 = corners[:, 1]
-    x1 = corners[:, 2]
-    y1 = corners[:, 3]
+    x0 = corners[:, :, 0]
```

Comment:
can we instead do `[..., 0]`?  Does that work or cause issues?

Reply:
yes it causes some errors as 
`x0 = corners[...,0] `gives shape **(None, num_cutouts, 4, 1)** 
`x0 = corners[:,:,0]` gives shape **(None, num_cutouts, 1, 1)**